site/components/Navigation/index.tsx

```diff
@@ -11,14 +11,19 @@ import { NavItemDesktop } from "./NavItemDesktop";
 import { NavItemMobile } from "./NavItemMobile";
 import { HeaderMobile } from "../Header/HeaderMobile";
 import { ChangeLanguageButton } from "components/ChangeLanguageButton";
-
+import * as styles from "./styles";
 // dynamic import for ThemeToggle as its reads the document and localStorage of Browser
 // see https://nextjs.org/docs/advanced-features/dynamic-import#with-no-ssr
 
 const ThemeToggle = dynamic(() => import("./ThemeToggle"), { ssr: false });
-
+// these types should be the same?
 type PageName = "Home" | "Buy" | "Resources" | "Disclaimer" | "Infinity";
-
+export type NavItem =
```

Comment:
`NavItem` is only used for `NavItemMobile` IDs

I would rename it to `NavItemMobileID`

Reply:
Thats a better name for sure. updated.